### DISCUSSION TITLE
9.2.2 version upgrade is giving error during deployment

### DISCUSSION THREAD

**Discussion by pchebro** [directus/directus#10506] [Archive]
```
simple	14-Dec-2021 15:36:35	2021-12-14T15:36.35-0500 [STG/0]      OUT info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
simple	14-Dec-2021 15:36:35	2021-12-14T15:36.35-0500 [STG/0]      OUT        [31;1m**ERROR**[0m Unable to build dependencies: exit status 1
simple	14-Dec-2021 15:36:35	2021-12-14T15:36.35-0500 [STG/0]      ERR Failed to compile droplet: Failed to run all supply scripts: exit status 14
```

Hi, I'm getting above error when deploying updated directus app with latest version 9.2.2 to development environment in cloud foundry. Please provide if you have any insight to this issue. 

### To Reproduce

Update Directus version and deploy to development environment through bamboo deployment plan.

### Errors Shown

```
simple	14-Dec-2021 15:36:35	2021-12-14T15:36.35-0500 [STG/0]      OUT info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
simple	14-Dec-2021 15:36:35	2021-12-14T15:36.35-0500 [STG/0]      OUT        [31;1m**ERROR**[0m Unable to build dependencies: exit status 1
simple	14-Dec-2021 15:36:35	2021-12-14T15:36.35-0500 [STG/0]      ERR Failed to compile droplet: Failed to run all supply scripts: exit status 14
```

**Comment by rijkvanzanten** [comment]
How are you deploying Directus exactly? How are you building what? Development environment?

**Comment by pchebro** [comment]
Updated the directus version in package.json. 
Deploying using bamboo specs. In specs I have code to run yarn install during build to install packages to the running cloud foundry environment.
Development environment is nothing but non prod environment in PaaS.

**Reply by rijkvanzanten** [reply]
Do you have a more specific example? I don't know what bamboo specs are, nor have ever used CloudFoundry. For what it's worth, we deploy built distributions to Dockerhub and npm, so you shouldn't have to build Directus itself. If it's crashing during the build of the dependencies, then its most certainly caused by the builder environment

**Reply by pchebro** [reply]
So I have a content management app using vue.js language to provision Directus UI. Bamboo specs is pipeline as code that has predefined tasks to deploy the package to our private cloud environment. As you mentioned this may not be the issue with Directus latest version but thought to see if you guys came across this issue before. Thanks for the prompt response.

**Reply by rijkvanzanten** [reply]
You're using Vue.JS to provision Directus? What does the pipeline do in order to deploy Directus? It sounds like you're building the npm packages "fresh" through a yarn install? Seeing CloudFoundry runs on top of Kubernetes (afaict), shouldn't you run Directus within a pre-built image that already includes the node dependencies?

**Reply by pchebro** [reply]
In our case, it will download all packages as part of build and zip as artifact for deployment. Ignore about vue.js I mentioned. We use that for custom code, not to provision Directus. Sorry for confusion.
